projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -3640,11 +3925,23 @@ function render() {
 		return apply_filters( 'grunion_contact_form_field_html', $rendered_field, $field_label, ( in_the_loop() ? get_the_ID() : null ) );
 	}
 
+	// phpcs:disable WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+
+	/**
+	 * Render the label.
+	 *
+	 * @param string $type - the field type.
+	 * @param int    $id - the ID.
+	 * @param string $label - the label.
+	 * @param bool   $required - if the field is marked as required.
+	 * @param string $required_field_text - the text in the required text field.
+	 *
+	 * @return HTML
```

Comment:
"HTML" isn't a type.
```suggestion
	 * @return string HTML
```
Same on all the rest below.

Reply:
10f3b43572f6dc46f89a56b976a721eb2218a1c6